com.gitlab.j0chn.nextcloud_password_client.json

```
@@ -71,7 +71,7 @@
                 "x86_64"
             ],
             "build-commands": [
-                "cp -r nextcloud_password_client /app/nextcloud_password_client",
+                "cp -r nextcloud_password_client/build/linux/x64/release/bundle /app/nextcloud_password_client",
```

```suggestion
                "cp -r nextcloud_password_client/linux/x64/release/bundle /app/nextcloud_password_client",
```

because of the way the tarball is made (and extracted).

---

I removed "build" from the path locally. ending in 
```
flatpak run com.gitlab.j0chn.nextcloud_password_client.json
Error: app/com.gitlab.j0chn.nextcloud_password_client.json/x86_64/master is not installed
```
or in
```
com.gitlab.j0chn.nextcloud_password_client
bwrap: execvp nextcloud_password_client: No such file or directory
```